src/applications/coronavirus-screener/components/FormResult.jsx

```diff
@@ -9,7 +9,7 @@ export default function FormResult({ formState }) {
   let resultClass = '';
   const [resultSubmitted, setResultSubmittedState] = React.useState({
     isSubmitted: false,
-    startTime: 0,
+    startTime: moment().unix(),
```

Comment:
@mchelen-gov  this starts the timer on page load.  I was going for starting it on first button click.

Reply:
oooh yeah good point

Reply:
Looking at previous code it is also starting timer on page load.  I thought I had it figured out to start with first button event but does not seem so.  This is another good reason to move state to multiQuestionForm.  should be able to do that pretty quick.

Reply:
@mchelen-gov  @Yuffster  I updated state initialization to happen in multiQuestionForm so it would be easier to initialize the analytics timer on first button click instead of page load.  This will probably complicate merge with the refactor in `https://github.com/department-of-veterans-affairs/vets-website/pull/12805 ` but seems worth it.  Please review!